src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -226,7 +238,29 @@ def _search(
                 query_kwargs.update(extra_query_kwargs)
 
             if query_kwargs["sort_by"] == "betterPriority":
-                query_kwargs["aggregate_kwargs"] = self.build_better_priority_sort_kwargs(request)
+                choice = None
+                if features.has(
+                    "organizations:better-priority-sort-experiment",
+                    organization,
+                    actor=request.user,
+                ):
+                    choice = expt_manager.get(
+                        "PrioritySortExperiment", org=organization, actor=request.user
+                    )
+                # force into variant1 for internal testing
+                if features.has(
+                    "organizations:issue-list-better-priority-sort",
+                    organization,
+                    actor=request.user,
+                ):
+                    choice = "variant1"
+
+                if choice == "baseline":
+                    query_kwargs["sort_by"] = "date"
+                else:
+                    query_kwargs["aggregate_kwargs"] = self.build_better_priority_sort_kwargs(
```

Comment:
@ceorourke do we need this if `organizations:issue-list-better-priority-sort` is true as well?

Reply:
Yeah? What do you mean? In the function that builds the kwargs it set the appropriate kwargs for variant1